README.md

```diff
@@ -59,44 +102,61 @@ This list is for websites, services, software, tools and more: everything that y
+- [Strange Parts](https://strangeparts.com) - [YouTube channel](https://www.youtube.com/channel/UCO8DQrSp5yEP937qNqTooOw) about electronics, manufacturing, making, world travel, living in and making things in China.
+
 ## Documentation
 - [Inkscape Electric Symbols](https://github.com/upb-lea/Inkscape_electric_Symbols) - Circuit Drawing Symbols for Inkscape
 - [Tabula](http://tabula.ondata.it/) - Extract tabular data from a pdf, very useful for extracting pin tables or part characteristics from datasheets.
 - [WebPlotDigitizer](https://automeris.io/WebPlotDigitizer/) - Extract data from plots, charts, etc., very useful for getting part performance curves from datasheets.
 - [WaveDrom](https://wavedrom.com/) - Create waveforms and timing diagrams from a JSON description file.
 
-## Simulators
-
-### Analog and Mixed Signal Circuit Simulators
-
-- [LTspice](https://www.analog.com/en/design-center/design-tools-and-calculators/ltspice-simulator.html) - The industry standard free SPICE circuit simulator from Linear Technologies. Also see the unofficial [LTwiki](http://ltwiki.org/?title=Main_Page) and [Group](https://groups.io/g/LTspice).
-- [ngspice](http://ngspice.sourceforge.net/) - Open source SPICE circuit simulator.
-- [Circuit JS/Falstad](http://www.falstad.com/circuit/circuitjs.html) - Free, open source online simulator with electron flow visualization (rewrite of original Java applet by Paul Falstad).
-- [EveryCircuit](https://everycircuit.com) - Free to try online, visual, interactive circuit simulator for simpler circuits.
-- [Qucs](http://qucs.sourceforge.net/) - Open source, cross-platform, non-SPICE-based circuit simulator, with with S-parameter and Harmonic Balance capability.
-- [Qucs-S](https://ra3xdh.github.io/) - Open source fork of Qucs using SPICE for simulation.
-- [QucsStudio](http://qucsstudio.de/) - Free, closed-source, Windows-only fork of Qucs with a similar interface, new engine, and more features.
-- [Open Circuit Design Software](http://opencircuitdesign.com) - Open Source, full EDA suite chip design suite, focused on keeping up with commercial tools.
-- [TINA-TI](http://www.ti.com/tool/TINA-TI) - Exclusive [DesignSoft-TINA](https://www.tina.com) build for Texas Instruments, bundled with Texas Instruments Models.
-- [CppSim](https://www.cppsim.com/) - Free, open source circuit simulator that leverages the C++ language to achieve very fast simulation times.
-- [Scilab with Xcos](https://www.scilab.org/) - Free, open source numerical computing alternative to MATLAB. Xcos provides Electrical System modeling capability similar to Simulink.
-- [iCircuit](http://icircuitapp.com/) - Easy to use electronic circuit simulator, its advanced simulation engine can handle both analog and digital circuits and features realtime always-on analysis.
-- [Micro-Cap](http://www.spectrum-soft.com/download/download.shtm) - Professional-grade mixed signal simulator with wide variety of interactive simulation types.
```

Comment:
The [Spectrum Software](https://en.wikipedia.org/wiki/Spectrum_Software) ( the micro-cap developer) went down and micro-crap is therefore obsolete, AFAIK. Let me know any divergence.

Reply:
Let's keep them as references/history, maybe adding a parenthesis signaling it's discontinued/obsolete.

Reply:
@joaoantoniocardoso Agreed.

Reply:
I strongly disagree with marking this as obsolete. Macro-cap is still a best-in-class simulator that works without any need to be attached to Spectrum Soft. I would call this software "complete" (if we're being generous), or "unmaintained" (if we are being accurate) rather than "obsolete". 

"Obsolete", by definition, means that it is "no longer in use" or "Outmoded in design, style, or construction". Neither of these attributes are true for macro-cap